src/database/migrations/20200213061003-create-room.js

```diff
@@ -6,18 +6,53 @@ export default {
       primaryKey: true,
       type: Sequelize.INTEGER
     },
-    type: {
-      type: Sequelize.STRING,
-      allowNull: false
-    },
     accommodation_id: {
       type: Sequelize.INTEGER,
+      allowNull: false,
+      onDelete: 'CASCADE',
+      references: {
+        model: 'Accommodation',
+        key: 'id',
+        as: 'accommodation_id',
+      },
+    },
+    wifi: {
+      type: Sequelize.STRING,
       allowNull: false
     },
     booked: {
       type: Sequelize.BOOLEAN,
       default: false
     },
+    air_condition: {
```

Comment:
air_conditioner and wifi fields are not needed. The amenities field should be able to have all that since it is an array